packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -195,16 +195,14 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
         fileMetadata={message['attachedFilesMetadata'] || []}
         downloadHandler={fileDownloadHandler}
         /* @conditional-compile-remove(file-sharing) */
```

Comment:
add conditional compile for @conditional-compile-remove(teams-inline-images-and-file-sharing) here as well